# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: architecture/powerpc64/package/*/ps3-gelic-skb-alloc.patch
# Copyright (C) 2020 - 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-5.14/mm/page_alloc.c.vanilla	2021-09-09 14:02:38.785182874 +0200
+++ linux-5.14/mm/page_alloc.c	2021-09-09 14:03:05.162181822 +0200
@@ -338,7 +338,11 @@
 
 int min_free_kbytes = 1024;
 int user_min_free_kbytes = -1;
+#ifdef CONFIG_PPC_PS3
+int watermark_boost_factor __read_mostly;
+#else
 int watermark_boost_factor __read_mostly = 15000;
+#endif
 int watermark_scale_factor = 10;
 
 static unsigned long nr_kernel_pages __initdata;
